client/uploadScrambles.js

```diff
@@ -105,12 +105,11 @@ function extractJsonFromZip(filename, zipId, pw, cb) {
         } else {
           cb("Wrong password");
         }
-        return;
-      } else {
-        throw error;
       }
+      console.error("Meteor.call() error: " + err);
```

Comment:
We're both making "Let's not spend extra effort" arguments.

To minimize this effort I changed the code to your version.